### PR TITLE
Fixed the copy to clipboard on Android + added sample

### PR DESCRIPTION
**Pull request by warrenbenyahia**
- Added a new sample : CopyToClipboard (TextBox + Button (copy text))
- Fixed the clipboard for android (it wasn't making the copy)

GitHub Issue (If applicable): #

## PR Type
What kind of change does this PR introduce?

Bugfix

New sample

## What is the current behavior?

Copying a text in the clipboard wasn't working (Android only)

## What is the new behavior?

It now copies the text in the clipboard

## PR Checklist

Please check if your PR fulfills the following requirements:

~~- [ ] Tested code with current [supported SDKs](../README.md#supported)~~
~~- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)~~
~~- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)~~
~~- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)~~
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

https://nventive.visualstudio.com/Umbrella/_workitems/edit/35178

### CODE REVIEW

File: src/Uno.UWP/ApplicationModel/DataTransfer/Clipboard.Android.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ public static partial class Clipboard
 	{
 		public static void SetContent(DataPackage content)
 		{
-			if (content?.Text == null)
+			if (content?.Text != null)
```
Comment (jeromelaban) [inline src/Uno.UWP/ApplicationModel/DataTransfer/Clipboard.Android.cs]: There's also the issue that resetting the content of the clipboard does not work.

Comment (warrenbenyahia) [reply src/Uno.UWP/ApplicationModel/DataTransfer/Clipboard.Android.cs]: @jeromelaban I checked and putting null just make the app crash. But setting the text to empty reset the clipboard

File: src/Uno.UI.Wasm/WasmScripts/Uno.UI.js [hunk 2]
```diff
@@ -971,6 +971,9 @@ var Uno;
                 const shouldRaiseLoadEvents = WindowManager.isLoadEventsEnabled
                     && this.getIsConnectedToRootElement(childElement);
                 parentElement.removeChild(childElement);
+                // Mark the element as unarranged, so if it gets measured while being
+                // disconnected from the root element, it won't be visible.
```
Comment (MatFillion) [inline src/Uno.UI.Wasm/WasmScripts/Uno.UI.js]: c'est bizarre que ce fichier soit présent dans ta PR

File: src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Input/Models/CopyToClipboardViewModel.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+namespace UITests.Shared.Windows_UI_Xaml_Input.Models
+{
+	[Bindable]
+    public class CopyToClipboardViewModel : ViewModelBase
+	{
+		public CopyToClipboardViewModel(CoreDispatcher dispatcher) : base(dispatcher)
+		{
+			Copy = CreateCommand(ExecuteCopy);
+			//Text = string.Empty;
```
Comment (MatFillion) [inline src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Input/Models/CopyToClipboardViewModel.cs]: I think this is not needed

**Pull request merged**